test/blackbox-tests/test-cases/bisect-ppx/run.t

```diff
@@ -0,0 +1,37 @@
+Normal bisect_ppx behavior
+  $ cd bisect-pps
+  $ dune exec --root . ./test.exe
```

Comment:
The `--root .` is superfluous here. When `dune` is executed inside `dune`, it disables root detection and systematically cwd as the root.